### PR TITLE
1.0.0 release pivot

### PR DESCRIPTION
**Pull request by fsheremetyeva**
@brandonbrown please review my pull request. Thank you.

### CODE REVIEW

File: docs/Spec.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+---
+
+### Feature Definitions
+
+[Filter of GitHub Features](https://github.com/fsheremetyeva/capstone/labels/Feature)
+
+- Ability to create user volunteer account
+- Ability to create non-profit organization account
+- Ability to search for volunteer opportunities
+- Initial Dashboard for volunteer account
```
Comment (brandonbrown) [inline docs/Spec.md]: What are the capabilities of this dashboard? We can see the following being feature possibilities:

- [ ] Update email
- [ ] Update Password
- [ ] Add/Modify/Remove Capabilities
- [ ] Add/Modify/Remove Availability
- [ ] Add new contact information
- [ ] Remove contact information
- [ ] Modify contact information
- [ ] Upcoming Promised Volunteer Events
- [ ] Volunteer History
- [ ] Certificates

File: docs/Spec.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+- Initial Dashboard for non-profit account
```
Comment (brandonbrown) [inline docs/Spec.md:40]: What are the capabilities of this dashboard? We can see the following being feature possibilities:

- [ ] Add/Update/Remove Volunteer Opportunities
- [ ] Upcoming Events
- [ ] Recently Completed Events
- [ ] New Volunteers

**Review by brandonbrown — CHANGES_REQUESTED** [review]
We would like to see a bit more detail with individual details. Most of these can be moved to your issues, and a few pre-existing issues have comments to help offer direction. On your issues, a short user story will be helpful for us to understand the context of the feature/feature group. For example:

"As a new volunteer, I can search this service to find volunteer opportunities fitting my interests."

The dashboards are the areas with the highest need for clarity. Once these issues have been created, you'll likely be ready for official approval.

File: docs/Spec.md (hunk 1 above)
Comment (fsheremetyeva) [reply docs/Spec.md]: Thank you, I moved this to my existing issue and commented there. 

A user is able to add/edit and delete information:

- [ ] Update email
- [ ] Update Password
- [ ] Add/Modify/Remove Capabilities
- [ ] Add new contact information
- [ ] Remove contact information
- [ ] Modify contact information
- [ ] Volunteer History

Other Features:
- [ ] Pie chart summarizing volunteer activity
- [ ] Generate Certificate

File: docs/Spec.md (hunk 2 above)
Comment (fsheremetyeva) [reply docs/Spec.md:40]: At this point only the first item is a part of my plan. But if the time permits, I will explore the other options. Thanks.

Comment (fsheremetyeva) [reply docs/Spec.md:40]: Added to Project Spec

File: docs/Spec.md (hunk 1 above)
Comment (fsheremetyeva) [reply docs/Spec.md]: Added to Project Spec

**Review by brandonbrown — APPROVED** [review]

**Pull request merged**